en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-configuration.md

```diff
@@ -0,0 +1,36 @@
+# Setting up the Environment 
+
+The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis stands for remote dictionary server. Redis store /serve that stores data as key-value pairs and this key-value store can be used as a database.
+
+## Setting up the environment
+
+Before you start configuring the Redis connector, you also need WSO2 MI, and we refer to that location as <PRODUCT_HOME>.
+
+To configure the SMPP connector, copy the following client libraries from the given locations to the `<PRODUCT_HOME>/lib` directory.
+
+* [ jedis-2.1.0.jar](https://mvnrepository.com/artifact/redis.clients/jedis/2.1.0)
+
+## Setting up the Redis server 
+
+1. Navigate to the [Redis server](http://redis.io/download) and download it. Please follow the steps given in this page to install the Redis server in your local machine.
+2. After setting up the **Redis Server** navigate to Redis installed location and execute **src/redis-server** command.
```

Comment:
./src/redis-server.sh ? The current does not look like a command.

Reply:
Updated.